### ISSUE TITLE
Support Rails 6.1.0

### ISSUE THREAD

**Issue by nahrivera7** [jmazzi/crypt_keeper#187]
Anyone working on upgrading to support Rails 6.1.0?
```
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    crypt_keeper (~> 2.1.0) was resolved to 2.1.0, which depends on
      activesupport (>= 4.2, < 6.1)

    rails (~> 6.1.0) was resolved to 6.1.0, which depends on
      activesupport (= 6.1.0)
```

**Comment by jesperronn** [comment]
Same problem for me.

Maintainers (@jmazzi ?) please reach out if you could use help to provide PR supporting 6.1

**Comment by nikz** [comment]
Working on this here: https://github.com/nikz/crypt_keeper 

Looks like it needs an update to the `pg` gem which seems to break a few things.

**Comment by Chew** [comment]
Thank you for your work. Sadly the maintainer looks to be AWOL. Let's hope he sees the PR and merges it.

To nikz: 
![image](https://user-images.githubusercontent.com/8278263/102825604-b8429180-43a4-11eb-854f-2945f0ae5016.png)
Thank you for your work :)

**Comment by nikz** [comment]
PR is here: https://github.com/jmazzi/crypt_keeper/pull/188

Feedback appreciated.

**Comment by jmazzi** [comment]
Resolved in #188
